src/pathpyG/core/HigherOrderGraph.py

```diff
@@ -12,34 +12,31 @@
 
 from pathpyG.utils.config import config
 from pathpyG.algorithms.temporal import temporal_graph_to_event_dag
-
+from pathpyG.core.IndexMap import IndexMap
+from pathpyG.core.HigherOrderIndexMap import HigherOrderIndexMap
 
 # TODO: Add description for arguments
 class HigherOrderGraph(Graph):
     """HigherOrderGraph based on torch_geometric.Data."""
 
-    def __init__(self, paths: PathData, order: int = 1, node_id: Any = None, **kwargs: Any):
+    def __init__(self, paths: PathData, order: int = 1, **kwargs: Any):
         """Generate HigherOrderGraph based on a given PathData instance.
 
         Args:
             paths:
             order:
-            node_id:
             **kwargs:
 
         Example:
             ```py
             import pathpyG as pp
 
             paths = pp.PathData()
+            paths.mapping = pp.IndexMap(['a', 'b', 'c', 'd'])
             paths.add_walk(torch.Tensor([[0, 1, 2], [1, 2, 3]]))
-            g2 = Graph(paths, k=2, node_id=['a', 'b', 'c', 'd'])
+            g2 = Graph(paths, k=2)
```

Comment:
Shouldn't this be `HigherOrderGraph(...)`?

Reply:
fixed